thinc/__init__.py

```diff
@@ -3,3 +3,11 @@
 
 from .about import __version__
 from .config import registry
+
+
+# fmt: off
+__all__ = [
+    "registry",
+    "__version__"
```

Comment:
For future editing/diffs, add trailing commas to all of the lists, or in the cases where you don't really need the `fmt` tags, you can just format them with black.